### ISSUE TITLE
fails to get keyboard focus in some (all?) tiling window managers

### ISSUE THREAD

**Issue by nbirnel** [derf/feh#162]
Behavior:
Using dwm or awesome window managers, and st or xterm terminal emulators,
run feh -F a_folder_of_images/*
Feh displays the first image.
Hit n to advance to next image. The "n" is passed to the terminal. Feh does nothing.
Hit Ctrl-c to kill feh. Feh closes.

Bizarre workaround:
run urxvt. (Optionally, close urxvt immediately.)
from st or xterm, launch feh again. Feh receives keyboard focus as expected.

In openbox WM, feh works fine with st or xterm.

Tested on:
feh version 2.10-19-gbf418d2
Compile-time switches: curl exif stat64 xinerama 

and on:
feh version 2.10-18-g2790131-dirty
Compile-time switches: curl xinerama 

Considering this is an issue using two different WMs and two different TERMs, I suspect
this may be a bug in feh, not the other software.

**Comment by bradenbest** [comment]
I found a [workaroud](http://superuser.com/questions/868583/mwm-hints-inconsistently-working-not-working-in-i3/873654#873654) for the time being.

**Comment by Inaimathi** [comment]
Using `feh` with `xmonad` here. The workaround I've settled on is just adding `spawn "urxvt -e exit"` to my `onStartup` routine, which seems to work perfectly.

**Comment by camilleg** [comment]
For `feh` in `awesome`, append to `/etc/xdg/awesome/rc.lua` the line `awful.util.spawn_with_shell("urxvt -e exit")`.

**Comment by ubone** [comment]
2018 report in dwm

`feh -F dir/` from `st` works normal

`Exec=feh -F %U` in `feh.desktop`:
- works from pcmanfm-qt and [firefox](http://ftp.mozilla.org/pub/firefox/releases/61.0.1/)
- doesn't work from pcmanfm (gtk2)

workaround: `Exec=st feh -F %U` in `feh.desktop`
